src/com/example/entrypointprint/UserRegistrationApp.java

```diff
@@ -0,0 +1,81 @@
+import java.util.Scanner;
+
+public class UserRegistrationApp {
+
+    // Method to register a new user
+    public static void registerUser() {
+        Scanner scanner = new Scanner(System.in);
+
+        // Prompt for user input
+        System.out.println("Enter first name:");
+        String firstName = scanner.nextLine();
+
+        System.out.println("Enter last name:");
+        String lastName = scanner.nextLine();
+
+        System.out.println("Enter username:");
+        String username = scanner.nextLine();
+
+        System.out.println("Enter date of birth (YYYY-MM-DD):");
+        String dateOfBirth = scanner.nextLine();
+
+        // Create a new User object with the entered data
+        User user = new User(firstName, lastName, username, dateOfBirth);
+
```

Comment:
Kindly check your work. In the previous merged PR. The User constructor was taking in a dateOfBirth of type Date, and now you scanning a String and passing it in the Date object.
![Screen Shot 2024-05-31 at 4 42 41 PM](https://github.com/Percy256/UserRegistration-model/assets/83646778/3b030bb9-656a-428c-84c1-564dbc357d48)

public User(String firstName, String lastName, String username, Date dateOfBirth);

- I also asked you to ignore the .class files in the previously approved PR